src/main/java/br/ufpe/cin/mergers/util/CSDiffRunner.java

```diff
@@ -0,0 +1,171 @@
+package br.ufpe.cin.mergers.util;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.common.io.CharStreams;
+
+import br.ufpe.cin.app.JFSTMerge;
+import br.ufpe.cin.exceptions.TextualMergeException;
+import br.ufpe.cin.files.FilesManager;
+
+public final class CSDiffRunner {
+    public static String runCSDiff(CSDiffScript script, String leftContent, String baseContent, String rightContent) throws TextualMergeException {
+        try {
+            File leftFile = createContributionFile("left", leftContent);
+            File baseFile = createContributionFile("base", baseContent);
+            File rightFile = createContributionFile("right", rightContent);
+            File outputFile = FilesManager.createTempFile("output");
+
+            runCSDiff(script, leftFile, baseFile, rightFile, outputFile);
+            String output = FilesManager.readFileContent(outputFile);
+
+            String leftFilePath = leftFile.getAbsolutePath();
+            String baseFilePath = baseFile.getAbsolutePath();
+            String rightFilePath = rightFile.getAbsolutePath();
+
+            return fixConflictMarkers(output, leftFilePath, baseFilePath, rightFilePath);
+            // return output;
+        } catch (IOException e) {
+            throw new TextualMergeException("Error during opening of temporary output file");
+        }
+    }
+
+    private static File createContributionFile(String name, String content) throws TextualMergeException {
+        try {
+            File file = FilesManager.createTempFile(name);
+            FilesManager.writeContent(file.getAbsolutePath(), content);
+            return file;
+        } catch (IOException e) {
+            throw new TextualMergeException("Error during opening of temporary input file(s)");
+        }
+    }
+
+    private static void runCSDiff(CSDiffScript script, File leftFile, File baseFile, File rightFile, File outputFile) throws TextualMergeException {
+        try {
+            File scriptFile = createScriptFile(script);
+            String[] command = buildCSDiffCommand(scriptFile, leftFile, baseFile, rightFile, outputFile);
+    
+            Process process = Runtime.getRuntime().exec(command);
```

Comment:
@leonardoAnjos16 isso roda em qualquer sistema operacional? Lembro que a versão do Jonatas não.

Reply:
@guilhermejccavalcanti realmente, só roda em Linux, esqueci de comentar isso. Pra rodar em outros sistemas, precisaria implementar o algoritmo do CSDiff em Java mesmo.

Reply:
@pauloborba  @leonardoAnjos16 então eu sugiro algum IF aqui que verifica o OS, se não for linux, usa uma estratégia default (Diff3) e printa que tá usando o Diff3 em vez do Csdiff pro conta do OS.

Reply:
concordo @guilhermejccavalcanti . ou melhor simplesmente avisar que essa opção de linha de comando (consecutive lines) só está disponível para linux, ou windows com aquela extensão de linha de comandos

Reply:
Certo, vou fazer isso!